lib/caeser.rb

```diff
@@ -0,0 +1,67 @@
+require 'pry'
+class Caeser
+
+  def initialize(shift_value)
+    @alphabet = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"]
+    @to_cipher_key = @alphabet.rotate(-shift_value)
+    @from_cipher_key = @alphabet.rotate(shift_value)
+  end
+
+
+  def eng_to_cipher(phrase)
+    # take given phrase and split into letters and spaces
+    letters = phrase.split("")
+    # create new array of ciphered letters by
+      # getting index in alphabet
+        # If " " return " "
+        # else return same index from rotated alphabet (key
+    ciphered_letters = letters.map do |letter|
+      if letter == " "
```

Comment:
I want you to think about how could make this work for all non alphabet characters, i.e. if we had commas, apostrophes, etc.